FGEGraphics/GraphicsHelpers/Renderer.cs

```diff
@@ -365,6 +404,81 @@ public void RenderRectangle3D(Matrix4 mat)
         GL.BindVertexArray(0);
     }
 
+
+
+    /// <summary>
+    /// Renders a Circle based on Location (Vectors Normalized Down)
+    /// </summary>
+    /// <param name="center">The Location for the circle to be rendered</param>
+    /// <param name="radius">Radius of the circle</param>
+    /// <param name="view">View to render Circle in</param>
+    /// <param name="color">Color of the circle</param>
+    /// <param name="enableCullFace">Enabling Cull Face makes both sides of the Circle Render</param>
+    /// <param name="enableDepthTest">Enabling Depth Testing allows depth checking of the pixel depth already stored</param>
+    /// <param name="depthFunc">Specifies the condition for passing the depth test
+    /// <param name="depthMask">Controls whether the depth buffer is writable or not
+    /// <param name="enableBlending">When blending is enabled, OpenGL mixes the color of the incoming fragment (new pixel) with the color already on the screen (old pixel)</param>
+    /// <param name="srcBlend">Color of the new fragment</param>
+    /// <param name="dstBlend">Color already on the screen</param>
+    public void RenderGroundCircle(
+        Location center,
+        float radius,
+        View3D view,
+        Vector4 color = default,
+        bool enableCullFace = false, 
+        bool enableDepthTest = true,
+        bool depthMask = true,
+        DepthFunction depthFunc = DepthFunction.Lequal,
+        bool enableBlending = true,
```

Comment:
you don't want this blend/cull/etc. stuff in here, the caller should modify when needed. Also this should not be a `Ground` circle, just a generic circle. Engine code should be generic.